digitalfabrik/entitlementcard
# Proposal: english store descriptions

@f1sh1918  This is a slight overhaul of #1382 . Just a proposal with a slightly less hacky workaround (and some refactorings :grimacing:)

### Short description

Since the proposal for store translations was not accepted by nuernberg, the english translations will be simply added with line breaks to the existing store description

### Proposed changes

- add possibility to import "RabattEN" and "RabattDE" columns in csv import (if the old column "Rabatt" is present, then this column will be used for the german text, while the english text stays empty) 
- both translations are merged in the acceptance store pipeline
- For the csv export, the originally imported entries are used to write out the translations separately
- Refactored CSV import and export to use header names instead of hardcoded indices
- Remove unused ID column from written CSV

### Side effects

The backend should be deployed first, then the new data should be deployed (this PR is backward compatible).

### Resolved issues



Fixes: #1343 

### Notes

- open pr of the data repository can be found here:
https://github.com/digitalfabrik/sozialpass-nuernberg-data/pull/11

### Testing
- run "Import data" from backend runConfigs (dev only)
- check store descriptions in app
